app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,68 @@
+class VotesController < ApplicationController
+  before_action :review, only: [:create, :update]
+  before_action :park, only: [:create, :update]
+
+  def create
+    if params[:vote] == "up"
+      vote = @review.votes.new(vote: 1)
```

Comment:
Check out the rest of the comments. You should create a `vote_params`. This will turn to `vote = @review.votes.new(vote_params)`